problemsets-mat223/linearalbegra.tex

```diff
@@ -2974,24 +3010,51 @@ \section*{Linear Transformations}
 		Let $L:V\to W$ be a transformation and let $X\subset V$ be a set.
 		The \emph{image of the set $V$ under $L$}, denoted $L(V)$, is the set
 		\[
-			L(V)=\{\vec x: \vec x=L(\vec y)\text{ for some }\vec y\in V\}.
+			L(V)=\Set{\vec x\in W \given \vec x=L(\vec y)\text{ for some }\vec y\in V}.
 		\]
 	\end{definition}
 
 	\question
-	Let $S=\{\mat{x\\y}:0\leq x,y\leq 1\}\subseteq \R^2$ be the filled-in unit square
-	and let $C=\{\vec 0, \vec e_1,\vec e_2,\vec e_1+\vec e_2\}\subseteq \R^2$ be the corners of the 
-	unit square.
+	Let $S=\Set*{\mat{x\\y} \given 0\leq x,y\leq 1}\subseteq \R^2$ be the filled-in unit 
+	square and let $C=\Set{\vec 0, \vec e_1,\vec e_2,\vec e_1+\vec e_2}\subseteq \R^2$
+	be the corners of the unit square.
 	\begin{parts}
 		\item Find $\mathcal R(C)$, $W(C)$, and $T(C)$ (where $\mathcal R$, $W$,
 			and $T$ are from the previous question).
-		\item Draw $\mathcal R(S)$, $T(S)$, and $\mathcal P(S)$ (where $\mathcal R$, $T$,
-			and $\mathcal P$ are from the previous question).
-		\item Suppose that $\ell=\{\text{ all convex combinations of }\vec a\text{ and }\vec b\}$
-			is a line segment with endpoints $\vec a$ and $\vec b$
-			and $A$ is a linear transformation. Must $A(\ell)$ be a line segment? What are
-			its endpoints?
-		\item Explain how images of sets relate to the \emph{Italicising N} task.
+			\begin{solution}
+				$\mathcal R(C) = \Set{\vec 0,\vec e_2,-\vec e_1,-\vec e_1+\vec e_2}$.
+
+				$W(C) = C$.
+
+				$T(C) = \Set*{\mat{2\\0},\mat{3\\0},\mat{2\\1},\mat{3\\1}}$.
+			\end{solution}
+		\item Draw $\mathcal R(S)$, $T(S)$, and $\mathcal P(S)$ (where $\mathcal R$,
+			$T$, and $\mathcal P$ are from the previous question).
+			\begin{solution}
+				XXXX
+			\end{solution}
+		\item Suppose that $\ell=\Set{\text{all convex combinations of }\vec a\text{ and }\vec b}$
+			is a line segment with endpoints $\vec a$ and $\vec b$ and $A$ is a 
+			linear transformation. Must $A(\ell)$ be a line segment? 
+			What are its endpoints?
+			\begin{solution}
+				Yes, $A(\ell)$ must be a line segment.
+				Its endpoints are $A(\vec a)$ and $A(\vec b)$.
+
+				If $\vec x=\alpha_1\vec a+\alpha_2\vec b$ is a convex combination
+				from $\ell$, then $A(\vec x)=\alpha_1A(\vec a)+\alpha_2A(\vec b)$,
+				which is a convex combination of $A(\vec a)$ and $A(\vec b)$.
```

Comment:
Discuss the degenerate case.

Reply:
Do you mean the case in which a=b?

Reply:
Well, `A` could be the zero transformation, so `A(a)` to `A(b)` would be a degenerate segment, for example.

Reply:
Oh, I see. I'll make a note about what happens if A(a) = A(b), such as with the zero transformation.